catalog/transformer/catalog-transformer-streaming-impl/pom.xml

```diff
@@ -64,17 +61,24 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform.util</groupId>
+            <artifactId>platform-util</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-xml</artifactId>
+            <version>9.5</version>
```

Comment:
Can we define this version as a property in a parent pom and use it here and in the other pom?

Reply:
Done